### ISSUE TITLE
manufacturer data in scan response overwrites the data in the advertisement packet

### ISSUE THREAD

**Issue by stmariej** [IanHarvey/bluepy#338]
When scanning, dev.getScanData() returns "Manufacturer" from the scan response packet ONLY.  

I believe it should append the scan response data to the advertisement packet OR have 2 keys (Manufacturer Adv & Manufacturer SR) ---- not overwrite it.

**Comment by BernardoSlailati** [comment]
That's true. I'm with this problem here. Can I know if the received data is coming from an advertising or scan response?
